dashboards/grafana-dashboard-insights-rosocp-general.configmap.yaml

```diff
@@ -1,19 +1,19 @@
 apiVersion: v1
 data:
-  Playground-1684824280869.json: |-
+  ROSOCP-1696939187132.json: |-
     {
       "__inputs": [
```

Comment:
Instead of `__inputs` variables, could you make use of templating variable (i.e `datasource`) that already exists at [line-695](https://github.com/RedHatInsights/ros-ocp-backend/pull/134/files#diff-f5f74135dc85fa6a7af9d78b6c004648ba6ac703750db803d375c8ef9d7abe90R695)?
Refer - https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/monitoring.md#setting-the-correct-datasource 

It will replace the value of datasource on change. I doubt that current dashboard panels loads fresh data on top datasource value selection.

Reply:
I am not sure of `__inputs` or the whole json structure. I have basically created the dashboard in grafana and exported it which created this JSON. As PR is raised 2months back I am not 100% sure if data is loaded based on top datasource value. For now I will change the `Playground-1684824280869` to `ROSOCP-1696939187132`.